cpp/src/spatial/hausdorff.cu

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright (c) 2019-2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you mpoint_a_y not use this file except in compliance with the License.
```

Comment:
```suggestion
 * you may not use this file except in compliance with the License.
```
Find and replace of `a` and `b` went haywire here :rofl:

Reply:
YES. I missed `a_` as `a`, but thought I found all of the errors pretty quick so didn't redo it. Lesson learned. :roll_eyes: 

Thanks!